core/templates/domain/utilities/language-util.service.ts

```diff
@@ -163,7 +163,7 @@ export class LanguageUtilService {
     return language ? language.description : null;
   }
 
-  getLanguageDirection(languageCode: string): string {
+  getLanguageDirection(languageCode: string | null): string {
```

Comment:
why is this change needed? I don't see any specific usecase of `null` in the function's implementation

Reply:
We're using the function in translation-language.service.ts file. There `activeLanguageCode` has been passed as an argument to this function. So this `activeLanguageCode` can be null, therefore we will need to make the type null here.

Reply:
Update it to not use null at all, leaving the variable undefined.